config/docker.cfg

```diff
@@ -0,0 +1,17 @@
+[shoobx:mocks3]
+log-level = INFO
+directory = /shoobx/shoobx.mocks3/var/mock_s3
+hostname = localhost
+reload = True
+debug = False
+
+[shoobx:server]
+host-ip = 0.0.0.0
+host-port = 8081
+
+[shoobx:cors]
+Access-Control-Allow-Origin = localhost:8081
```

Comment:
Hm, this this actually work? It looks like it is configured to allow requests from `localhost:8081`, but the server itself already runs on `localhost:8081`. Am I missing something or this declaration is not useful?

Reply:
Yeah, you are right, it not healthy for docker. I made POC for the app and with this configuration it will work with shoobx app (because of `socat`)